pkg/scheduler/scheduler.go

```diff
@@ -563,7 +562,7 @@ func (s *Scheduler) requeueAndUpdate(log logr.Logger, ctx context.Context, e ent
 		e.requeueReason = queue.RequeueReasonFailedAfterNomination
 	}
 	added := s.queues.RequeueWorkload(ctx, &e.Info, e.requeueReason)
-	log.V(2).Info("Workload re-queued", "workload", klog.KObj(e.Obj), "clusterQueue", e.ClusterQueue, "queue", klog.KRef(e.Obj.Namespace, e.Obj.Spec.QueueName), "added", added, "status", e.status)
```

Comment:
status is already logged here https://github.com/kubernetes-sigs/kueue/blob/93dce4be3eb2450bbe46d5e1911d208853960625/pkg/scheduler/scheduler.go#L143-L147

Reply:
`requeueReason`